test/smoke_test/bcda_client.go

```diff
@@ -136,6 +139,30 @@ func writeFile(resp *http.Response) {
 	}
 }
 
+func isValidNdjsonFile(filename string) bool {
```

Comment:
The Go team would tell you to name this `isValidNDJSONFile` https://github.com/golang/go/wiki/CodeReviewComments#initialisms